group/init/schema.js

```diff
@@ -15,9 +15,10 @@ module.exports = {
     groupKey: { $ref: '#/definitions/groupKey' },
     tangles: {
       type: 'object',
-      required: ['group'],
+      required: ['group', 'members'],
```

Comment:
:fire: this is a breaking change, takes this schema to v2 I think. See other parts of this repo (I think) for how I've handled versioning

Reply:
ah you want compat with ahau. so i'll just add a `version: 'v2'` field ok?

Reply:
added `version: 'v2'`

Reply:
and moved to a v2 dir